app/views/layouts/application.html.erb

```diff
@@ -6,30 +6,57 @@
 
     <title><%= content_for?(:title) ? yield(:title) : "Caesar" %></title>
 
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css"
```

Comment:
By the way, PFE does use maxcdn for some external resources (the icon font mostly, I think.) It can lead to icons not displaying if that request gets blocked or times out for some reason.